bin/download_an_all.py

```diff
@@ -0,0 +1,103 @@
+import sys
+from threading import Thread
+# To run this example, you need an API key
+# available from https://api.ecmwf.int/v1/key/
+
+import cdsapi
+c_ml = cdsapi.Client()
+c_pl = cdsapi.Client()
+c_pv = cdsapi.Client()
+c_tl = cdsapi.Client()
+c_sfc = cdsapi.Client()
+
+def ml():
+    c_ml.retrieve('reanalysis-era5-complete', {
+        'class': 'od',
+        'time': sys.argv[2],
```

Comment:
please assign argv parameters to variables and use those later in the code, e.g.
```
date, time = sys.argv[1:3]
types = sys.argv[3:]
```

Reply:
see argparse

Reply:
True, but might be overkill here.